src/commands/assign.js

```diff
@@ -207,7 +235,11 @@ async function checkAssigned() {
         }
         // Add the id of the newly assigned submission to the list of assigned
         // submissions.
-        assigned.push(s.id);
+        assigned.push({
+          submission_id: s.id,
+          project_id: s.project.id,
+          timeToFinish: (12 - moment.utc().diff(s.assigned_at, 'hours')),
```

Comment:
Excellent addition. But will this update on every `setPrompt()`? It looks like it's only set once and then never updated again.

You will need to store `s.assigned_at` to use in `setPrompt()`.